### ISSUE TITLE
got less detection results and lower confidence level to original darknet

### ISSUE THREAD

**Issue by adonishong** [qqwweee/keras-yolo3#11]
Highly appreciate for qqwweee's work. ^_^

With same cfg(coco, 80 classes, yolov3.cfg), and same weights to darknet yolov3 (yolov3.weights), but the detection results and confidence level are much lower than darknet version. With Python 2.7, tensorflow_gpu 1.7, keras 

original version result:
person: 99%
person: 99%
person: 99%
person: 97%
person: 94%
person: 92%
person: 90%
person: 79%
person: 78%
person: 66%
person: 66%
person: 65%
person: 65%
person: 59%
person: 58%
person: 51%

results from keras-yolo3:

(416, 416, 3)
Found 10 boxes for img
('person 0.48', (798, 518), (1028, 951))
('person 0.48', (1092, 33), (1234, 263))
('person 0.54', (1416, 155), (1633, 643))
('person 0.56', (1312, 19), (1470, 347))
('person 0.73', (1011, 771), (1273, 1077))
('person 0.85', (413, 697), (663, 1068))
('person 0.86', (1550, 892), (1722, 1071))
('person 0.88', (1369, 155), (1513, 634))
('person 0.89', (1632, 123), (1798, 587))
('person 0.96', (1355, 704), (1552, 1009))

here is the image for testing

![t1](https://user-images.githubusercontent.com/9166212/38789168-04f220b6-416b-11e8-900f-fefbb63259ef.jpg)

**Comment by adonishong** [comment]
with latest version (9ef8628), lower confidential level issue still exists...

**Comment by ahhan02** [comment]
I met the problem too, @adonishong 
Have you solve it???

**Comment by arkjiang** [comment]
same problem here. @ahhan02 

In darknet output 
vehicle: 21%
vehicle: 32%
vehicle: 37%
vehicle: 58%
people: 43%

In converted Keras of same network 

Found 3 boxes for img
vehicle 0.04 (0, 280) (74, 421)
vehicle 0.16 (132, 33) (327, 133)
vehicle 0.22 (48, 114) (229, 304)

Missing people object and all vehicle confidence rate drops.

**Comment by okathira** [comment]
In my case today (compiled exe file [darknet_yolo_v3](https://github.com/AlexeyAB/darknet/releases/tag/darknet_yolo_v3) and custom objects), the problem was due to different resizing processes.

Instead of letterbox resizing, resize the image directly. (ref: https://stackoverflow.com/questions/58269867/why-darknet-model-to-keras-model-conversion-drops-accuracy)
Then adjust the boxes to aspect ratio.
my changes: https://github.com/okathira/keras-yolo3/commit/c3aa2d3a9b0bebdeee43ac4ff7423604c4af7846

I solved it this way, but there may be a better option.
